shared/lib_irradproc.cpp

```diff
@@ -305,41 +312,8 @@ void incidence(int mode,double tilt,double sazm,double rlim,double zen,double az
 				}
 			else          /* For other than vertical axis */
 				{
-				arg = sin(zen)*sin(azm-xsazm)/
-						( sin(zen)*cos(azm-xsazm)*sin(xtilt) + cos(zen)*cos(xtilt) );
-				if( arg < -99999.9 )
-					rot = -M_PI/2.0;
-				else if( arg > 99999.9 )
-					rot = M_PI/2.0;
-				else
-					rot = atan(arg);
-								/* Put rot in II or III quadrant if needed */
-				if( xsazm <= M_PI )
-					{
-					if( azm > xsazm && azm <= xsazm + M_PI )
-						{     /* Ensure positive rotation */
-						if( rot < 0.0 )
-							rot = M_PI + rot;   /* Put in II quadrant: 90 to 180 deg */
-						}
-					else
-						{     /* Ensure negative rotation  */
-						if( rot > 0.0 )
-							rot = rot - M_PI;   /* Put in III quadrant: -90 to -180 deg */
-						}
-					}
-				else        /* For xsazm > pi */
-					{
-					if( azm < xsazm && azm >= xsazm - M_PI )
-						{     /* Ensure negative rotation  */
-						if( rot > 0.0 )
-							rot = rot - M_PI;   /* Put in III quadrant: -90 to -180 deg */
-						}
-					else
-						{     /* Ensure positive rotation */
-						if( rot < 0.0 )
-							rot = M_PI + rot;   /* Put in II quadrant: 90 to 180 deg */
-						}
-					}
+				rot = truetrack(azm*180/M_PI, zen*180/M_PI, tilt, sazm);
+				rot *= M_PI/180;
```

Comment:
The sin and cos difference identities show that the deleted definition of `arg` above is equivalent to the approach in the new `truetrack` function.  No quadrant checking necessary in the new approach because of the use of the `atan2` function, but I didn't verify that these quadrant checks are equivalent to what `atan2` does.